### PR TITLE
Desktop - Add more tooltips

### PR DESCRIPTION
**Pull request by JeremyMoeglich**
This PR adds tooltips to many buttons that didn't have them before.
List of locations where tooltips were added in the format {location}: {tooltip}

1. Sidebar - Settings icon: "Settings"
2. Sidebar - Jobs icon next to Settings: "Recent Jobs"
3. ExplorerTopBar - Key Icon: "Unlock"
4. ExplorerTopBar - Three Lines: "File Display Options"
5. ExplorerTopBar - Inspector Toggle: Either "Show Inspector" or "Hide Inspector"
6. Library Settings - Database Icon: "TODO" What will this button do? It doesn't do anything currently
7. Library Settings - Edit Icon: "Edit Location"
8. Library Settings - Trash Can: "Delete Library"
9. Location Settings  - Trash Can: "Delete Location"
10. Location Settings - Retry Icon: "Rescan Location"
11. Tags Settings - Trash Can: "Delete Tag"

### CODE REVIEW

File: packages/interface/src/components/explorer/ExplorerTopBar.tsx [hunk 1]
```diff
@@ -265,20 +265,22 @@ export const TopBar: React.FC<TopBarProps> = (props) => {
...
-									<Key className={TOP_BAR_ICON_STYLE} />
-								</TopBarButton>
-								// </Tooltip>
-							}
-						>
-							<div className="block w-[350px]">
-								<KeyManager className={TOP_BAR_ICON_STYLE} />
-							</div>
-						</OverlayPanel>
+						<Tooltip label="Unlock">
```
Comment (brxken128) [inline packages/interface/src/components/explorer/ExplorerTopBar.tsx:268]: this should say "Key manager" :)

**Review by brxken128 — CHANGES_REQUESTED** [review]
looks pretty good! i'll give this a spin up when i can just to confirm they all work, iirc there were some oddities with the items in the top bar and getting tooltips to show correctly, but i may be misremembering.

thank you!

File: packages/interface/src/components/explorer/ExplorerTopBar.tsx (hunk 1 above)
Comment (JeremyMoeglich) [reply packages/interface/src/components/explorer/ExplorerTopBar.tsx]: Done,
I'm still trying to understand many parts of the repo might try to make the add tag button in the inspector work next.
Certainly some of the best code Ive seen

File: packages/interface/src/components/explorer/FileItem.tsx [hunk 2]
```diff
@@ -43,6 +44,7 @@ function FileItem({
 }: Props) {
 	const objectData = data ? (isObject(data) ? data : data.object) : null;
 	const isVid = isVideoExt(data.extension || '');
+	const fullName = `${data.name ?? ''}${data.extension ? `.${data.extension}` : ''}`;
```
Comment (Brendonovich) [inline packages/interface/src/components/explorer/FileItem.tsx:47]: Not a fan of this, utilising JSX's ability to ignore `undefined` and `null` is much nicer than ternaries with empty strings imo

**Review by Brendonovich — CHANGES_REQUESTED** [review]
very helpful, thanks for this!

File: packages/interface/src/components/explorer/FileItem.tsx (hunk 2 above)
Comment (JeremyMoeglich) [reply packages/interface/src/components/explorer/FileItem.tsx]: Changed it back to how it was before

**Review by brxken128 — APPROVED** [review]
thanks for the changes! finally got around to testing them and i'm happy with them - they all seem to be working great.

**Review by Brendonovich — APPROVED** [review]

**Pull request merged**